crates/ef-testing/src/evm_sequencer/account/v0.rs

```diff
@@ -64,3 +82,33 @@ impl KakarotAccount {
         })
     }
 }
+
+/// Analyze bytecode to build a jump map.
+/// author: REVM <https://github.com/bluealloy/revm/blob/main/crates/interpreter/src/interpreter/analysis.rs#L50>
+fn analyze(code: &Bytes) -> Vec<usize> {
```

Comment:
I don't think it makes sense for us as we don't need all these extra types etc, we just want to return an array of jumpdest indexes to write in storage.

Reply:
doesn't really use extra types right? Just avoids us copy pasting code in case it changes. You can just wrap the `Bytes` into a `Bytecode::LegacyRaw` and then analyse it and pick the jump table out of it. IMO makes more sense than having code duplication